### ISSUE TITLE
filecachetools deprecated

### ISSUE THREAD

**Issue by infowolfe** [peak6/Rocket.Chat.Audit#3]
https://github.com/xi/filecachetools is deprecated and no longer available in pip yet it's referenced in your code and in requirements.txt.

I'm not sure if this project is even maintained anymore, but updating requirements.txt to match the git syntax would be a step in the right direction for new installs.

**Comment by codyaray** [comment]
You're right that filecachetools is deprecated. I no longer work for this company so I don't have commit access anymore. But PRs are hopefully still welcome if you want to prepare one.

PS - No need to be passive aggressive. We're all doing the best we can here in the free and open source software world.

**Comment by infowolfe** [comment]
Ok? Sorry if you thought I was being passive aggressive, but I wasn't intending to be. This project seemed pretty dead and I genuinely wasn't expecting a response at all, let alone this fast.

More than anything I was attempting to find out if there was a current maintainer at peak6, but it seems that the original maintainer (you) no longer works for the company so there probably isn't.

Please note that this bug (again) isn't intended for you personally, but for peak6 as an org.

**Comment by codyaray** [comment]
I doubt they've had to do anything with it since I left. It was a pretty set-and-forget kind of thing. So they probably just haven't run into issues, tbh.

It sucks that filecachetools is deprecated. Do you know what the recommended upgrade path is for filecachetools or if there's a replacement library?

IIRC I just used filecachetools to make the logged data cleaner across restarts. Plus some DB optimizations that aren't totally necessary.
https://github.com/peak6/Rocket.Chat.Audit/blob/master/rocketchat.audit.py#L51

So worst case, just use cachetools here and accept that edits during restarts may not have all the data (room name or editedBy fields), if that fits for your use case.
